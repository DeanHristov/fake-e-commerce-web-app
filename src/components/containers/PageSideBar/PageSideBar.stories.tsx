import type { Meta, StoryObj } from '@storybook/react';
import PageSideBar, { IPageSideBarProps } from './index';

const meta: Meta<IPageSideBarProps> = {
  title: 'Containers/PageSideBar',
  component: PageSideBar,
  parameters: {
    layout: 'centered',
  },
  argTypes: {},
};

export default meta;

type Story = StoryObj<IPageSideBarProps>;

export const Primary: Story = {
  render: () => <PageSideBar />,
};
