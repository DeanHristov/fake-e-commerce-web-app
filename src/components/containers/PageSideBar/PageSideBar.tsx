import { FC } from 'react';

export interface IPageSideBarProps {}

const PageSideBar: FC<IPageSideBarProps> = ({}) => {
  return <div />;
};

export default PageSideBar;
