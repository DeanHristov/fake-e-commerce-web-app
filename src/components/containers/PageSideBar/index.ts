import PageSideBar, { IPageSideBarProps } from './PageSideBar';

export type { IPageSideBarProps };

export default PageSideBar;
