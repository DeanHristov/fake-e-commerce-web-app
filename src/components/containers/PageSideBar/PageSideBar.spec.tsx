import { render } from '@testing-library/react';
import PageSideBar from './index';

describe('Containers/PageSideBar <PageSideBar {...}>', () => {
  it('Should render default state', () => {
    const { container } = render(<PageSideBar />);

    expect(container).toMatchSnapshot();
  });
});
