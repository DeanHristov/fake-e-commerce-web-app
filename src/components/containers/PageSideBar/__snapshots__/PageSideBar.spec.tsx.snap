// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Containers/PageSideBar <PageSideBar {...}> Should render default state 1`] = `
<div>
  <div />
</div>
`;
